css/items-section.css

```diff
@@ -44,17 +63,57 @@
 
 .item img {
     width: 100%;
+    height: 100%;
+    object-fit: cover;
 }
 
 .item p {
-    font-size: var(--font-size-extra-small);
+    font-size: var(--font-size-small);
 }
 
-.empty-container-text {
-    display: block;
-    text-align: center;
+.buttons-container {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 35%;
```

Comment:
I'd remove the `height`. Usually it's better to allow the content to dictate the element' height. Try using padding instead.